manager/container.go

```diff
@@ -63,6 +63,7 @@ type containerData struct {
 	loadAvg                  float64 // smoothed load average seen so far.
 	housekeepingInterval     time.Duration
 	maxHousekeepingInterval  time.Duration
+	allowedHouseKeepingGap   time.Duration
```

Comment:
It was dropped while testing via cli argument.
At the moment, it would be better to set as a constant.
I set 1 minute as a default in housekeeping().